meilisearch-http/src/routes/tasks.rs

```diff
@@ -169,7 +170,7 @@ async fn get_tasks(
 
 async fn get_task(
     meilisearch: GuardedData<ActionPolicy<{ actions::TASKS_GET }>, MeiliSearch>,
-    task_id: web::Path<TaskId>,
+    task_id: Option<web::Path<TaskId>>,
```

Comment:
this is actually very strange. It works, but it's more like a ugly hack. If you really want to catch any kind of overflow, we will want to parse it to a `String` first, and then try to parse a `TaskId` from that

```suggestion
task_id: web::Path<String>,
```

Reply:
I learned this [here](https://github.com/actix/actix-web/discussions/1986) 🤔